examples/rl/workflows/policy_manager/policy_manager.py

```diff
@@ -4,7 +4,7 @@
 import sys
 from os.path import dirname, realpath
 
-from maro.rl.policy import LocalPolicyManager, MultiNodePolicyManager, MultiProcessPolicyManager
+from maro.rl.policy import LocalPolicyManager, MultiNodePolicyManager, MultiNodeDistPolicyManager, MultiProcessPolicyManager
```

Comment:
This name sounds a little strange. @Jinyu-W  Any ideas for a better name?

Reply:
We can go through all the modes first and declare what situation the mode is for and rename them if necessary